lib/puppet/indirector/ldap.rb

```diff
@@ -62,13 +62,13 @@ def ldapsearch(filter)
   # Create an ldap connection.
   def connection
     unless @connection
-      raise Puppet::Error, "Could not set up LDAP Connection: Missing ruby/ldap libraries" unless Puppet.features.ldap?
+      raise Puppet::Error, _("Could not set up LDAP Connection: Missing ruby/ldap libraries") unless Puppet.features.ldap?
```

Comment:
Could you add a translator note that `ruby/ldap libraries` refers to missing application dependencies?